modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -253,6 +257,26 @@ class SeqTranslate(site: Site, systems: Systems, settings: Settings) {
     } )
   )
 
+  def resumeObserve(seqId: Sequence.Id)(seqState: Sequence.State): Option[Process[Task, Event]] = {
+    val observeIndex: Option[(ObserveContext, Int)] = seqState.current.execution.zipWithIndex.find(_._1.kind === ActionType.Observe).flatMap{ x => { x match {
```

Comment:
i think the `flatMap { x => x match` can be reduced to `flatMap { case ...`

Reply:
Oh, yes. I started with that, but then expanded things to catch a syntax error, and forgot to go back.